### DISCUSSION TITLE
[제안] 실험을 위한 mini SATRN

### DISCUSSION THREAD

**Discussion by jo-member** [bcaitech1/p4-ocr-hansarang#30] [Ideas]
현재 baseline의 backbone은 DenseNet으로 조금 무겁습니다
빠른 결과를 위해 논문에서 제시하고있는 shallow CNN으로 바꾸고
hidden dimension과 input size를 줄여서 빠른 실험결과를 봅시다!

**Comment by remaindere** [comment]
mini SATRN : 
network: mini_SATRN
input_size:
  height: 128
  width: 128
SATRN:
  encoder:
    hidden_dim: 300
    filter_dim: 600
    layer_num: 6
    head_num: 8
  decoder:
    src_dim: 300
    hidden_dim: 128
    filter_dim: 512
    layer_num: 3
    head_num: 8
Attention:
  src_dim: 512
  hidden_dim: 128
  embedding_dim: 128
  layer_num: 1
  cell_type: "LSTM"
  
shallow CNN코드도 올려주세요~~
